landing-page/content/services/data-compaction.html

```diff
@@ -1,7 +1,8 @@
 ---
-Title: Time Travel
-AsciinemaCast: time_travel.cast
-Category: Post
+Title: Data Compaction
+Description: Data compaction is supported out-of-the-box and you can choose from different rewrite strategies such as bin-packing, sorting, or z-ordering to optimize file layout and size.
```

Comment:
Z-ordering is not supported yet.

Reply:
Ah ok, removed in [b033a57](https://github.com/apache/iceberg-docs/pull/18/commits/b033a571fdb082c317a5987ed1ec5705d3bdce6c)